server/matchmaker/algorithm/team_matchmaker.py

```diff
@@ -325,17 +325,17 @@ def assign_game_quality(
         minority_bonus = min(minority_bonus, config.MINORITY_BONUS)
         rating_disparity = abs(match[0].cumulative_rating - match[1].cumulative_rating)
         unfairness = rating_disparity / config.MAXIMUM_RATING_IMBALANCE
-        deviation = statistics.pstdev(ratings)
-        rating_variety = deviation / config.MAXIMUM_RATING_DEVIATION
+        max_team_deviation = max(map(statistics.pstdev, [match[0].displayed_ratings, match[1].displayed_ratings]))
```

Comment:
Why are you using displayed ratings here?

Reply:
I was looking for the ratings separately for each team, I was not sure which is the correct value to use. Which value do you suggest to use?